client/src/electron/electron-builder.json

```
@@ -0,0 +1,33 @@
+{
+  "files": ["client/build/electron", "client/www", "client/resources/tray", "!node_modules/electron"],
+  "asarUnpack": ["client/output", "client/tools"],
+  "artifactName": "Outline-Client.${ext}",
+  "directories": {
+    "output": "client/output/build/dist"
+  },
```

Found a way to [customize the `BUILD_RESOURCES_DIR`](https://github.com/electron-userland/electron-builder/blob/fa6fc16040a93c6ee751a7a27ab6eeb1dbdd31a7/docs/api/programmatic-usage.md?plain=1#L48):

```suggestion
  "directories": {
    "buildResources": "client/build",
    "output": "client/output/build/dist"
  },
```